addons/l10n_id_efaktur_coretax/__init__.py

```diff
@@ -2,3 +2,19 @@
 
 from . import models
 from . import controllers
+
+from odoo import api, SUPERUSER_ID
+
+def _post_init_hook(cr, registry):
+    # When installed, should create the luxury tax group with specific XML ID
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+
+    if not env.ref("l10n_id.l10n_id_tax_group_luxury_goods", raise_if_not_found=False):
+        env['ir.model.data'].create({
+            "name": "tax_group_luxury_goods",
+            "module": "l10n_id",
```

Comment:
I missed that, sorry :grimacing: 
```suggestion
    if not env.ref("l10n_id.l10n_id_tax_group_luxury_goods", raise_if_not_found=False):
        env['ir.model.data'].create({
            "name": "l10n_id_tax_group_luxury_goods",
            "module": "l10n_id",
```
I believe here you will create a tax group with the wrong XML id

Reply:
ah right, missed this part as well